pkg/chunk/cache/mock.go

```diff
@@ -0,0 +1,45 @@
+package cache
+
+import (
+	"context"
+	"sync"
+)
+
+type mockCache struct {
```

Comment:
Could just call this an in memory cache instead of specifically a mock since it has real functionality.

Though if we are making sure no one uses it in the future, maybe mock is best.

Reply:
It currently doesn't ever remove entries, so best only used in tests.  The FIFOCache is for inmemory usage.

Reply:
👍